### PR TITLE
feat: after order user have tickets

### PR DESCRIPTION
**Pull request by Tsaijieming**
優化結帳流程及畫面

### CODE REVIEW

**Review by sethxd88 — APPROVED** [review]
ok

**Review by LaichuLai — COMMENTED** [review]
LGTM

**Review by LaichuLai — APPROVED** [review]
LGTM

File: app/views/dishes/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,86 @@
 <div class="bg-gray-100 max-w-screen-md container mx-auto pb-4">
 
+<<<<<<< HEAD
```
Comment (naaa7) [inline app/views/dishes/show.html.erb:3]: 沒刪到head

**Review by naaa7 — APPROVED** [review]
okok

**Pull request merged**